analysis/CompareNECRvsLength.py

```diff
@@ -0,0 +1,136 @@
+import SiemensQuadraProperties as sqp
```

Comment:
What is the purpose of this file?

Reply:
I wrote this code to compare NECR vs. Length curves for different crystals or different physics lists and would prefer to keep it on github

Reply:
Moved it to another directory that will be used for code development